### ISSUE TITLE
[Static Site] What's new page "Learn more" links accessible name needs to be unique

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#6442]
## Area or Component
Static Site
https://www.simplereport.gov/using-simplereport/whats-new/

## Quick Summary

The accessible name for the "Learn more" links on the "What's new" page should be unique for each "Learn more" link since each link goes to a different page but all say "Learn more".

## Accessibility Violation
https://docs.deque.com/issue-help/1.0.0/en/aria-name-missing-incorrect

## Action Requested
Add a unique accessible name for the "Learn more" links

## Screenshots (optional)
<img width="767" alt="Screenshot 2023-08-28 at 17 04 07" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/60cfad33-7a34-4804-9616-958b20ec8bf9">
